lib/auth/auth_with_roles.go

```diff
@@ -290,52 +348,18 @@ func (a *ServerWithRoles) GetActiveSessionTrackers(ctx context.Context) ([]types
 	}
 
 	var filteredSessions []types.SessionTracker
+	joinerRoles, err := a.authServer.GetRoles(ctx)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
```

Comment:
user := a.context.User
	joinerRoles, err := services.FetchRoles(user.GetRoles(), a.authServer, user.GetTraits())
	if err != nil {
		return nil, trace.Wrap(err)
	}
```suggestion
	user := a.context.User
	joinerRoles, err := services.FetchRoles(user.GetRoles(), a.authServer, user.GetTraits())
	if err != nil {
		return nil, trace.Wrap(err)
	}
```